activejob/lib/active_job/test_helper.rb

```diff
@@ -98,6 +98,24 @@ def queue_adapter_for_test
     #     end
     #   end
     #
+    # The number of times a unspecified job is enqueued can be asserted.
```

Comment:
This language seems wrong to me. How about "The number of times jobs were enqueued, except for a specific class"